status-updates/2023/2023-05-12.md

```diff
@@ -0,0 +1,41 @@
+# Friday, May 12, 2023 - 9:00am EEST
+
+## Participants
+
+Sergiu, Teo, RăzvanN, Liza, Vlad, Mihai
+
+## Discussions
```

Comment:
We can add what RN was saying about social media. Maybe we can take action into that direction.